website/pages/docs/plugins/sources/stripe/overview.md

```diff
@@ -0,0 +1,41 @@
+# Stripe Source Plugin
+
+import { getLatestVersion } from "../../../../../utils/versions";
+import { Badge } from "../../../../../components/Badge";
+
+<Badge text={"Latest: " + getLatestVersion("source", `stripe`)}/>
+
+The CloudQuery Stripe plugin pulls data from Stripe and loads it into any supported CloudQuery destination (e.g. PostgreSQL).
+
+## Authentication
+
+In order to fetch information from Stripe, `cloudquery` needs to be authenticated using a [secret API key](https://stripe.com/docs/keys) from your Stripe account. You can view and manage your API keys in [the Stripe Dashboard](https://stripe.com/login?redirect=/account/apikeys).
+
+# Configuration Reference
+
+This is the (nested) spec used by the Stripe source plugin:
+
+- `api_key` (string, required):
+  Your secret API key from the Stripe Dashboard.
+
+- `max_retries` (integer, optional. Default: 2):
+  Number of retries if a request was rate limited.
+
+## Example
```

Comment:
Nit: This example should probably move to the top of the page, as that's the thing you need first before you need further details.

Reply:
Good nit, fixed